index.html

```diff
@@ -3,8 +3,8 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="data.js"></script>
-  <script src="index.js"></script>
+  <script src="data.js" defer></script>
```

Comment:
In this case, you don't need the `defer` attribute on the data.js script tag - since the code in this file doesn't use any DOM elements, it's not necessary. Doesn't hurt to have it, but it's not needed. You *do* need it on the index.js script tag, since the code in that file needs access to DOM elements.